src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -1207,6 +1207,24 @@
     </Setting>
   </Provider>
 
+  <!--
+                                           ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                           ██ ███ █ ▄▄▄██ ▄▄▀█▄ ▄██ ▄▀▄ █▄ ▄██
+                                           ███ █ ██ ▄▄▄██ ▀▀▄██ ███ █ █ ██ ███
+                                           ███▄▀▄██ ▀▀▀██ ██ █▀ ▀██ ███ █▀ ▀██
+                                           ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Verimi" Documentation="https://docs.verimi.de/">
+    
+    <Environment Name="Production" Issuer="https://web.verimi.de/" />
+    
+    <Environment Name="Staging" Issuer="https://web.uat.verimi.cloud/" />
+
+    <Setting PropertyName="ClientCertificate" ParameterName="certificate" Type="Certificate" Required="true"
```

Comment:
Where is this setting used? As a general rule, settings are always defined here but you need custom code somewhere to resolve the setting and apply the required logic based on it.

Verimi's docs don't seem to imply TLS client authentication is supported for the OIDC integration (it's only for the mailbox stuff AFAICT) so it's likely you won't even need this. Should we need this, you would have to add a few lines inside this delegate:

https://github.com/openiddict/openiddict-core/blob/fea943370c614c26bd7152761415dde16ec7518b/src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationConfiguration.cs#L62-L78

Reply:
You are right. The certificate is only required for accessing the mailbox and later to communicate with their API, but it's not relevant for authentication, so I removed the setting here.